lib/src/manager_mixin.dart

```diff
@@ -33,4 +34,7 @@ mixin ManagerMixin implements Nyxx {
 
   /// A [GatewayManager] that manages gateway metadata for this client.
   GatewayManager get gateway => GatewayManager(this as NyxxRest);
+
+  /// A [ClientEmojiManager] that manages global emojis for this client.
+  ClientEmojiManager get emojis => ClientEmojiManager(this as NyxxRest, guilds.cache.values.map((g) => g.emojis));
```

Comment:
Again I don't think this is a good idea - this has multiple issues relating to caching.